.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+# command to install dependencies
+install:
+  - pip install sphinx codespell
+# command to run tests
+script:
+  - codespell --skip=".git,version-specific" --ignore-words-list=atleast --quiet-level=2; retVal=$?; if [ $retVal -eq 1 ]; then echo "Got one expected warning, so success!";  else $(exit $retVal); fi;
```

Comment:
What is the expected warning?

What if `retVal=0`, then we'll do an `exit 0`?

Reply:
The gymnastics here are because of the disabled suggestion because of the use of a proper name 
```
./docs/Cray-support.rst:19: Tim  ==> Time, tim  | disabled due to being a person's name
```
(see https://travis-ci.org/ocaisa/easybuild/jobs/496488317#L493). There was no way to turn this off (that I could find) and the return value is the number of messages it prints, so if this one is expected then we always see at least one, if it's 0 then even better.

Reply:
Maybe add a comment to clarify that?